sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -57,29 +57,48 @@ class CloudEvent(EventMixin):   #pylint:disable=too-many-instance-attributes
     :type source: str
     :param data: Event data specific to the event type.
     :type data: object
-    :param type: Required. Type of event related to the originating occurrence.
+    :keyword type: Required. Type of event related to the originating occurrence.
```

Comment:
should we remove the `Required` in the description?
```suggestion
    :keyword type: Type of event related to the originating occurrence.
```

Reply:
type is required and positional - it's not a keyword only